manifests/admin/params.pp

```diff
@@ -1,7 +1,7 @@
 # Pulp Admin Params
 class pulp::admin::params {
   $version            = 'installed'
-  $host               = 'localhost'
+  $host               = $::fqdn
```

Comment:
This change appears to be unrelated and I don't know pulp well enough to know if it makes a difference.